frontend/packages/dev-console/src/components/DefaultPage.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { ListPage } from '@console/internal/components/factory';
+import { ProjectsTable } from '@console/internal/components/namespace';
+import { PageHeading } from '@console/internal/components/utils';
+import './DefaultPage.scss';
+
+interface DefaultPageProps {
+  title: string;
+  children?: React.ReactNode;
+}
+const DefaultPage: React.FC<DefaultPageProps> = (props) => (
+  <div className="odc-default-page">
+    <PageHeading title={props.title}>{props.children}</PageHeading>
+    <hr
+      style={{
+        marginTop: '30px',
+        marginBottom: '30px',
+        borderBottom: '0.8px solid var(--pf-global--BackgroundColor--light-300)',
```

Comment:
Can we consider `$color-grey-background-border`(`#ccc`) for consistency with other pages? We should eventually align with PF4 palette, but we use `#ccc` elsewhere today.

Reply:
updated to #ccc